.github/workflows/release.yml

```diff
@@ -53,15 +53,22 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v1
+    - name: Check For setup.py
+      id: need-pypi
+      run: |
+        echo ::set-output name=setup-py::$( find . -wholename './setup.py' )
     - name: Set up Python
+      if: contains(steps.need-pypi.outputs.setup-py, 'setup.py')
```

Comment:
The issue with this is that a `setup.py` is created by the cookiecutter by default, so by default libs will deploy to pypi.